### PR TITLE
Added multiple contents example

### PR DESCRIPTION
**Pull request by javagl**
An example for `3DTILES_multiple_contents`

Preview: https://github.com/javagl/3d-tiles-samples/tree/add-multiple-contents/next/MultipleContents

### CODE REVIEW

File: next/MultipleContents/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  }
+});
+
+// Wait until the tileset is loaded, move it to a certain position
+// on the globe, and center the tileset in the viewer
+tileset.readyPromise.then(function() {
+  tileset.modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(
+    Cesium.Cartographic.toCartesian(
+      Cesium.Cartographic.fromDegrees(-75.1596759, 39.9509025, 0)));
+  viewer.zoomTo(tileset);
```
Comment (ptrgags) [inline next/MultipleContents/README.md:54]: You don't need to wait for the ready promise, `zoomTo()` already does this, see https://cesium.com/learn/cesiumjs/ref-doc/Viewer.html?classFilter=viewer#zoomTo

File: next/MultipleContents/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  if (Cesium.defined(innerContents) && innerContents.length === 2) {
+    trianglesContent = innerContents[0];
+    pointsContent = innerContents[1];
+  }
+});
+
+// Wait until the tileset is loaded, move it to a certain position
+// on the globe, and center the tileset in the viewer
+tileset.readyPromise.then(function() {
+  tileset.modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(
```
Comment (ptrgags) [inline next/MultipleContents/README.md:51]: You can set `modelMatrix` in the constructor.

File: next/MultipleContents/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+// The two different contents from the root tile. These are
+// read in the 'tileLoad' event listener, and stored here
+// for access from the Sandcastle UI
+var trianglesContent;
+var pointsContent;
+
+// Load the tileset from a local server
+var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'http://localhost:8003/tileset.json'
+}));
```
Comment (ptrgags) [inline next/MultipleContents/README.md:34]: Simplified the tileset setup code a bit and pointed out some CesiumJS idioms:

- moved the model matrix construction to the tileset creation time
- Used `Cartesian3.fromDegrees()` which is simpler than `Cartographic.toCartesian(Cartographic.fromDegrees())`
- moved the model a little bit into the air, it was clipping through the terrain by a little bit.
- `viewer.zoomTo()` can be called as soon as the tileset is constructed as I mentioned in another comment.

```suggestion
// Position the tileset on the globe.
var position = Cesium.Cartesian3.fromDegrees(-75.1596759, 39.9509025, 0.25);
var modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(position);

// Load the tileset from a local server
var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
    url : 'http://localhost:8003/tileset.json',
    modelMatrix : modelMatrix
}));
viewer.zoomTo(tileset);
```

**Review by ptrgags — COMMENTED** [review]
@javagl looks good, I just had a couple suggestions about the example code before we merge this.

**Comment by ptrgags** [comment]
Looks good, thanks @javagl!

**Comment by javagl** [comment]
The location was supposed the be the location of Cesium (the Company), based on the address. Some other samples seem to use a different location. This might be updated at some point, but is probably not critical :D

**Pull request merged**